build-system/tasks/e2e/puppeteer-controller.js

```diff
@@ -30,6 +25,13 @@ const {
 const {ControllerPromise} = require('./controller-promise');
 const {dirname, join} = require('path');
 
+const {
+  Browser, // eslint-disable-line no-unused-vars
+  JSHandle, // eslint-disable-line no-unused-vars
+  Page, // eslint-disable-line no-unused-vars
+  ElementHandle: PuppeteerHandle, // eslint-disable-line no-unused-vars
```

Comment:
Is there a way to avoid these?

Reply:
I could condense them, or else I could try to do some dynamic typing or just drop type enforcement.